### PR TITLE
Tests for TitleBar and TimeTimeSeriesSlider components

### LINKED ISSUE #188: Add Prettier and Husky Plugins
As developers our could should automatically prettify as a pre-commit action.

Context: https://github.com/onaio/gisida-react/pull/186#pullrequestreview-380421053

### PR DESCRIPTION
**Pull request by ciremusyoka**
Part of #185

Adds test for  `lib/components/TimeSeriesSlider` and `lib/components/TitleBar`.

### CODE REVIEW

File: test/lib/components/TimeSeriesSlider/TimeSeriesSlider.test.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+layer1['visible'] = true;
+
+const initialState = {
+	"map-1": {
+		primaryLayer: "education",
+		showFilterPanel: false,
+		layers: {"sample-layer": {...sampleLayer}, "education": {...layer1}},
+		timeseries: {}
+	},
+}
```
Comment (cKellyDesign) [inline test/lib/components/TimeSeriesSlider/TimeSeriesSlider.test.js]: @ciremusyoka Let's make sure the tabbing in this file is all set to 2 spaces, pls 🙏

File: test/lib/components/TitleBar/TitleBar.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const mockStore = configureStore()
+const store = mockStore(initialState);
+
+describe('TitleBar', () => {
+
+	it('renders without crashing', () => {
+    
+    shallow( 
+      <TitleBar  store={store} />
```
Comment (cKellyDesign) [inline test/lib/components/TitleBar/TitleBar.test.js]: Is there a tab in your component declaration?

**Review by cKellyDesign — CHANGES_REQUESTED** [review]
Let's tidy up the tabbing in these files please, they should look as nice in Github as they do in your IDE 💅

**Review by moshthepitt — CHANGES_REQUESTED** [review]
We should set up [prettier](https://prettier.io/) to deal with things like tabbing for us :)

Setting up prettier with a [husky](https://www.npmjs.com/package/husky) pre-commit hook takes about 15 minutes, and we can just copy the config from [other](https://github.com/onaio/reveal-frontend) [projects](https://github.com/onaio/grp-frontend) [that](https://github.com/onaio/kaznet-frontend) [have](https://github.com/onaio/js-tools) [this](https://github.com/opensrp/opensrp-web).

@cKellyDesign @ciremusyoka @kahummer ^^

**Comment by ciremusyoka** [comment]
@moshthepitt  i will add `prettier`.

**Comment by cKellyDesign** [comment]
@ciremusyoka what's the status of this PR?

**Pull request closed without merging**